assets/scripts/countdown.js

```diff
@@ -61,11 +63,13 @@ export default class Countdown {
             diff = targetTime - now,
             days = Math.floor( diff / ( 1000 * 60 * 60 * 24 ) ),
             hours = Math.floor( ( diff % ( 1000 * 60 * 60 * 24 ) ) / ( 1000 * 60 * 60 ) ),
-            minutes = Math.floor( ( diff % ( 1000 * 60 * 60 ) ) / ( 1000 * 60 ) );
+            minutes = Math.floor( ( diff % ( 1000 * 60 * 60 ) ) / ( 1000 * 60 ) ),
+            seconds = Math.floor( ( diff % ( 1000 * 60 ) ) / 1000 );
```

Comment:
Näissä ois varmaan voinut käyttää näitä laskettuja arvoja hyödyksi, esim. `days = Math.floor( ( diff % days ) / ( 1000 * 60 * 60 ) )` tjsp. Mutta ei tarvi nyt alkaa muuttamaan.